WolvenKit.App/Models/LaunchProfile.cs

```diff
@@ -16,9 +16,13 @@ public class LaunchProfile
     // install script files
 
     [Category("Install")]
-    [Display(Name = "Clean packed directory completely first")]
+    [Display(Name = "Clean packed directory before build?")]
```

Comment:
should that really ever be allowed?

Reply:
It's the current default behaviour :D

Reply:
eh okay, I guess we could remove that setting